src/test/java/io/strimzi/kafka/bridge/http/ConsumerTest.java

```diff
@@ -1301,6 +1317,7 @@ void createConsumerBridgeIdAndNameSpecified(VertxTestContext context) {
                     });
                     context.completeNow();
                 });
+        consumerService().deleteConsumer(context, groupId, "consumer-1");
```

Comment:
You have to wait on a `CompletableFuture` before deleting, see one of the comments already made above.
As usual, delete method on the next line.